bin/lesson_check.py

```diff
@@ -91,6 +91,17 @@
     'language-r',
     'language-shell',
     'language-sql',
+    'language-fortran',
```

Comment:
Good additions. Please alphabetize the list.

Reply:
done, thanks for the suggestion